src/driver.c

```diff
@@ -4893,7 +4894,8 @@ BOMULEUL CHAJARA SEGA ST-V  1997/04/11
 	DRIVER( mslider )	/* (c) 1997 Visco / Datt Japan */
 	DRIVER( ryorioh )	/* (c) 1998 Visco */
 	DRIVER( vasara )	/* (c) 2000 Visco */
-
+        DRIVER( vasara2 )	/* (c) 2001 Visco */
+        DRIVER( vasara2a )	/* (c) 2001 Visco */

```

Comment:
wrong indentation btw.

Reply:
Yeah i noticed that, i assumed it would have been cleaned up when merged i'll do it now